src/webgl-utils/track-context.js

```diff
@@ -336,3 +323,10 @@ export function popContextState(gl) {
   assert(gl.state);
   gl.state.pop();
 }
+
+export function setParameters(gl, params) {
+  assert(gl.state);
+  // Note: should we not pass gl.state.cache and let glSetParameters
```

Comment:
The reason I've been avoiding this is that would create a circular dependency between params and track-context.